error_check_logic.py

```diff
@@ -56,6 +55,21 @@ def Run(self, institution, session):
     self._Validate(class_groups, isValid_class_groups, 'Class Groups')
     self._Validate(student_groups, isValid_student_groups, 'Student Groups')
 
+    # Only call yaml.load if data is valid. And store the result
```

Comment:
Remember, we did this the easy way.
schemas.Dayparts() and so forth are functions that return a yayv.ByExample object.

But if we made them objects and somehow could keep them around (without being globals) we could store the loaded yaml with the object instead of modifying the datastore!